### PR TITLE
Review seb

### PR DESCRIPTION
**Pull request by sebastiandedeyne**
I added some sanitization to ensure slashes don't appear multiple times at the end of the directory path. Still needs a test to assert that `path` never contains trailing slashes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ $temporaryDirectory->path('dumps/datadump.dat'); // return  /full/path/to/tempor
 
 ### Deleting a temporary folder
 
-Once you're done processing your temporary data you can delete the entire temporary directory using the `delete` method. All files inside of it will be deleted.
```
Comment (sebastiandedeyne) [inline README.md]: Someone's editor isn't trimming trailing slashes 🤔 @AlexVanderbist @freekmurze

Comment (AlexVanderbist) [inline README.md]: For some reason .editorconfig has trim_trailing_whitespace set to false by default...

Comment (sebastiandedeyne) [inline README.md]: Ohhh my bad makes sense for markdown, since trailing spaces can have meaning. We can leave them trimmed here though.

**Comment by AlexVanderbist** [comment]
Thanks!

**Pull request merged**